ogidn/ogidn_default_parameters.json

```diff
@@ -1,6 +1,6 @@
 {
     "frisch": 0.4,
-    "g_y_annual": 0.038,
+    "g_y_annual": 0.048,
```

Comment:
@rickecon, Thanks for looking at this parameter value.  

Where are you getting this new number from?  I see the [Ministry of Finance](https://fiskal.kemenkeu.go.id/files/red/file/1691992285_fpa_red_july_2023.pdf) is forecasting 6-7% growth in GNI per capita over the next 20 years.